src/migrations/20191119153504-create-service.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('services', {
```

Comment:
I ask again, what is this table for?

Reply:
Yeah, @Mastel22 explained to me the idea about services, I missed the slack discussion, I am going to remove that.